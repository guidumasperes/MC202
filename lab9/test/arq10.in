257 27
1 J-378 1646
3 J-378
1 G-061 219
2 G-061
2 G-061
1 O-790 669
1 V-873 1668
1 B-583 689
4 1630
1 G-309 435
3 B-583
1 O-945 966
1 O-933 1837
1 Q-410 1480
1 A-536 1763
9
