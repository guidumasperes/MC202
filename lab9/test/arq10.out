chave removida!
219
219
wubba lubba dub dub!
chave removida!
